.github/workflows/update-gardenctl.sh

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env bash
 
-filteredTag=${1##*/}
-tag=${filteredTag:-v0.17.0}
-sha=${2:-8d33c751e8d32fe7fff15306c7de59cf15c45fb04e2f9abf988d3edd3f305cc4}
+tag=${1:-v0.17.0}
+mac_sha=${2:-f6cbd049d200a1857e9bb89cd614f0f96851ef2db307a18609a18effbb4497eb}
```

Comment:
what is this default sha for?

Reply:
again i have no preference on this default `mac_sha `, because i saw you have a default `sha` so i choose a default `mac_sha`, so just remove all the default value? thanks!